compiler/rustc_ast_lowering/src/expr.rs

```diff
@@ -229,6 +229,18 @@ impl<'hir> LoweringContext<'_, 'hir> {
                     },
                 ),
                 ExprKind::Await(expr, await_kw_span) => self.lower_expr_await(*await_kw_span, expr),
+                ExprKind::Use(expr, use_kw_span) => {
+                    if !self.tcx.features().ergonomic_clones() {
```

Comment:
Why is this being errored during ast lowering? It seems like it should just be gated during parsing like all other syntax features.

Reply:
Yeah, thanks. This is one of the things I know are not ok but did a quick solution to properly address later.

Reply:
Fixed